api/src/main/resources/workbench-api.yaml

```diff
@@ -931,6 +931,10 @@ paths:
           description: A workspace response containing workspace and access level
           schema:
             "$ref": "#/definitions/WorkspaceResponse"
+        403:
+          description: User doesnt have the correct access tier as workspace
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
```

Comment:
If this is a "copy" of terra rawls `GET /workspace` api, it returns more error codes. Not sure if they should be considered as well.
https://github.com/broadinstitute/rawls/blob/4e20d537bda3fdea3020a9ddfb05c47c8182eaf1/core/src/main/resources/swagger/api-docs.yaml#L3528

Reply:
No this is different from Terra, i have added an extra explicit check which throws 403 if the user has no CT access and the workspace is CT workspace. 
Otherwise terra just throws NOT_FOUND exception in that case